app/pubsub-listener.js

```diff
@@ -143,10 +143,21 @@ export default class PubsubListener {
         return
       }
 
-      let valid = await post.canShow(user.id)
+      if (!(await post.canShow(user.id))) {
+        return;
+      }
+
+      if (await post.isBannedFor(user.id)) {
+        return;
+      }
 
-      if (valid)
-        socket.emit(type, json)
+      const authorBans = await dbAdapter.getUserBansIds(post.userId)
+
+      if (authorBans.indexOf(user.id) >= 0) {
+        return;
+      }
```

Comment:
For the sake of clarity, can we move this piece to some subroutine, similar to `.isBannedFor`? Or maybe we have it somewhere already?